package.json

```diff
@@ -20,11 +21,14 @@
     "jsonwebtoken": "^8.3.0",
     "method-override": "^2.3.10",
     "methods": "^1.1.2",
-    "mongoose": "^5.2.2",
-    "mongoose-unique-validator": "^2.0.1",
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
+    "path": "^0.12.7",
```

Comment:
You don't need to install `path`. It comes bundled with `npm`.

Reply:
Ok that's cool. I did that in case of an upgrade.